Formula/lean.rb

```diff
@@ -45,6 +46,7 @@ def id' {α : Type} (x : α) : α := x
           split, repeat { assumption }
       end
     EOS
-    system "#{bin}/lean", testpath/"hello.lean"
+    system bin/"lean", testpath/"hello.lean"
+    system bin/"leanpkg", "--help"
```

Comment:
```suggestion
    system bin/"leanpkg", "help"
```
Looks like passing `--help` is unrecognised, and returns a non-zero exit code.

Reply:
Was just fixing that, sorry, you mean squash the 2 formulae changes? Or just squash the fixup I just did? If so think I did so, lemme know if you meant something else.